examples/control_tester.py

```diff
@@ -8,11 +8,12 @@
                                 MultiplyByConstantModN)
 from projectq.meta import Control
 from projectq.ops import (All, BasicMathGate, get_inverse, H, Measure, QFT, R,
-                          Swap, X)
-from projectq.meta._control import State
+                          Swap, X, CtrlAll, _command)

```

Comment:
Why is `_command` imported if not used?
```suggestion
                          Swap, X, CtrlAll)
```